src/notation/internal/notationinteraction.cpp

```diff
@@ -4651,6 +4651,7 @@ void NotationInteraction::addTimeSignature(Measure* measure, staff_idx_t staffIn
     startEdit();
     score()->cmdAddTimeSig(measure, staffIndex, timeSignature, true);
     apply();
+    doSelect({ measure }, SelectType::SINGLE, staffIndex);
```

Comment:
@Jojo-Schmitz This is my first approach at a fix.

a) Is it sensible?

b) How do I start to approach testing? The tests don't all pass locally on my machine, but it seems they do fine in the CI. 

Given that I'm not likely to be doing much more work on this after this fix, am I ok to test just by shoving commits at the CI and then we can squash them all afterwards? 

Otherwise I will need (likely several hours/days worth of) support from existing contributors to debug my local test setup and learn a bunch of new tooling

Reply:
Sirry, no idea, beyond my pay grade...